release of v10.72.3

tujoworker opened the pull request.

langz committed:
fix(CopyOnClick): display copy cursor for all children (#5011)

tujoworker committed:
fix(Forms): ensure Field.Selection with the autocomplete variant receives showIndicator in the first onType parameter object (#5012)

More
[info](https://dnb-it.slack.com/archives/CMXABCHEY/p1745592615795799).

langz commented:
Looks great, can we include https://github.com/dnbexperience/eufemia/pull/5011 as well?

langz commented:
Thanks 😃

tujoworker committed:
fix(Forms): ensure Field.Composition with horizontal layout will still wrap (#5007)

More
[info](https://dnb-it.slack.com/archives/CMXABCHEY/p1745577112603609?thread_ts=1745563078.030569&cid=CMXABCHEY).

langz commented:
Last time the tests in this release ran, they failed, hopefully it was just a one-time thing 🤞

tujoworker committed:
fix(CopyOnClick): copy `textContent` when given children is empty (#5014)

- Feature nr. 1 of [this
post](https://dnb-it.slack.com/archives/CMXABCHEY/p1745580508874319?thread_ts=1745578322.405229&cid=CMXABCHEY).
- PR
[Preview](https://eufemia-git-fix-copyonclick-textcontent-eufemia.vercel.app/uilib/components/copy-on-click/demos/#copy-text-content).
- We could almost always just relay on using `.textContent` instead of
trying to get the content of children. Any thoughts?

---------

Co-authored-by: Anders <anderslangseth@gmail.com>

tujoworker commented:
:tada: This PR is included in version 10.72.3 :tada:

The release is available on:
- [npm package (@latest dist-tag)](https://www.npmjs.com/package/@dnb/eufemia/v/10.72.3)
- [GitHub release](https://github.com/dnbexperience/eufemia/releases/tag/v10.72.3)

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:
